portfolio/src/main/webapp/index.html

```diff
@@ -16,6 +16,26 @@ <h1>My Portfolio</h1>
       <p>I have a great passion for programming since high school</p>
       <p>I've been to many programming competitions where I usually won prizes!</p>
 
+      <p>I have worked on many projects, most recent two are:</p>
+      <ul>
+          <li>
+            <b>QuizGame</b> 
+            <p>- A project for the OOP class, made with C++20 using latest features and a looot of OOP.</p>
+            <p>- Three types of questions (MultipleChoice, Word and Number) you have to answer.</p>
+            <p>- You can also add questions to the game.</p>
+            <p>- Can be found <a href="https://github.com/PredaMihaiDragos/Facultate/tree/master/Anul%201/POO/QuizGame">here</a></p>
+          </li>
+          <li>
+            <b>Website for blood analysis lab</b>
+            <p>- A project for both the Web class and my mother who has a blood analysis lab. </p>
+            <p>- Frontend: we were not allowed any APIs or to use any template, so it's plain HTML / CSS (with flexbox and grid) / JS written from 0. </p>
+            <p>- Backend: node.js </p>
+            <p>- Looks fairly good, has things like modal boxes, regex field validation, local storage, lots of AJAX and JS events, pages generated from templates etc</p>
+            <p>- Completely responsive on mostly any resolution!</p>
+            <p>- Can be found <a href="https://github.com/PredaMihaiDragos/Facultate/tree/master/Anul%201/TW/Proiect">here</a></p>
```

Comment:
As this project is in Romanian, I'd suggest to mention it somewhere. Maybe just in parens next to the link: Can be found here (note, it is in Romanian)